gazebo_ros/test/test_sim_time.cpp

```diff
@@ -88,7 +88,7 @@ TEST_F(TestSimTime, TestClock)
       EXPECT_GT(rclcpp::Time(msgs[i]->clock), rclcpp::Time(msgs[i - 1]->clock)) <<
         rclcpp::Time(msgs[i]->clock).nanoseconds() << "    " <<
         rclcpp::Time(msgs[i - 1]->clock).nanoseconds();
-      EXPECT_GT(times[i], times[i - 1]) << times[i].nanoseconds() << "    " <<
+      EXPECT_GE(times[i], times[i - 1]) << times[i].nanoseconds() << "    " <<
```

Comment:
I don't know how to ensure that test clock callback is executed after rclcpp internal clock callback, so I changed the condition of sim time test.

Reply:
I'm not sure the test is well-formed since it depends on the order in which the executor processes callbacks. It might be better to just assert that messages are received on the clock topic instead of also trying to test that the node's use_sim_time logic is working (seems out of scope for this test). So, I'd be okay with just removing this check.

Reply:
@jacobperron I removed the test of use_sim_time logic at https://github.com/ros-simulation/gazebo_ros_pkgs/pull/1380/commits/1864f19f02fae83dfbfc777bb8e14cf5570ec4dd.